client/package.json

```diff
@@ -32,7 +32,8 @@
 		"eject": "react-scripts eject",
 		"lint": "eslint ./src/**/*.{ts,tsx}",
 		"checks": "npm run test && npm run lint",
-		"format": "prettier --write './src/**/*.{ts,tsx}' --config ../.prettierrc"
+		"format": "prettier --write './src/**/*.{ts,tsx}' --config ../.prettierrc",
+		"prepare": "cd .. && husky install && cd client"
```

Comment:
when we get our apps monorepoized this will be simplified to `husky install` because we'll have a shared package.json in the root. but for now...